app/role.go

```diff
@@ -63,7 +67,10 @@ func getNamespaceRolesBatch(ctx context.Context, client authservice.AuthServiceC
 	return res.Roles, nil
 }
 
-func getAccountRole(ctx context.Context, client authservice.AuthServiceClient, actionGroup string) (*auth.Role, error) {
+func getAccountRole(ctx context.Context, client authservice.AuthServiceClient, actionGroup string, allowNone bool) (*auth.Role, error) {
+	if allowNone && strings.ToLower(strings.TrimSpace(actionGroup)) == accountActionGroupNone {
```

Comment:
suggestion: i'd rename the `allowNone` to `allowUnspecified` and remain consistent to with the enum definition.

Reply:
Unspecified is different than none. In this case the user has no actual account role, it's not that they have an account role of unspecified.